json/src/main/java/org/teavm/flavour/json/tree/Node.java

```diff
@@ -32,9 +32,12 @@
     @JSBody(params = { "node" }, script = "return typeof node == 'string';")
     static native boolean isString(Node node);
 
-    @JSBody(params = { "node" }, script = "return node === null;")
+    @JSBody(params = { "node" }, script = "return node === null || node === undefined;")
     static native boolean isNull(Node node);
 
+    @JSBody(params = { "node" }, script = "return node === undefined;")
```

Comment:
IMO, it's better to use `void 0` instead of `undefined`.

Reply:
done!